### PR TITLE
Assign CCEs to SSH permission checks

### PR DESCRIPTION
**Pull request by shawndwells**
Assign CCEs to rules added in https://github.com/ComplianceAsCode/content/pull/4812

### CODE REVIEW

File: linux_os/guide/services/ssh/file_owner_sshd_config/rule.yml [hunk 1]
```diff
@@ -12,6 +14,11 @@ rationale: |-
 
 severity: medium
 
+identifiers:
+    cce@rhel6: 82900-2
+    cce@rhel7: 82899-6
+    cce@rhel8: 82998-8
```
Comment (ggbecker) [inline linux_os/guide/services/ssh/file_owner_sshd_config/rule.yml]:
```suggestion
    cce@rhel8: 82898-8
```
Invalid CCE, just a typo.

File: shared/references/cce-redhat-avail.txt [hunk 2]
```diff
@@ -677,12 +677,3 @@ CCE-82890-5
...
 CCE-82893-9
-CCE-82894-7
-CCE-82895-4
-CCE-82896-2
-CCE-82898-8
-CCE-82899-6
-CCE-82900-2
-CCE-82901-0
-CCE-82902-8
-CCE-82903-6
```
Comment (redhatrises) [inline shared/references/cce-redhat-avail.txt]: @shawndwells why have so many CCEs been removed?

Comment (shawndwells) [reply shared/references/cce-redhat-avail.txt]: Three rules with three CCEs each.

Comment (redhatrises) [reply shared/references/cce-redhat-avail.txt]: hahahahaha.... more coffee is needed for me!

**Comment by redhatrises** [comment]
ACK

**Pull request merged**